lib/modules/manager/custom/regex/strategies.ts

```diff
@@ -89,7 +89,7 @@ function processRecursive(parameters: RecursionParameter): PackageDependency[] {
     regexes,
     config,
   }: RecursionParameter = parameters;
-  // abort if we have no matchString anymore
+  // abort if we have no matched strings anymore
```

Comment:
I'm not sure to change this.
@rarkins @HonkingGoose WDYT?

Reply:
No, revert